lib/main.js

```diff
@@ -29,8 +29,8 @@ class PythonLanguageClient extends AutoLanguageClient {
   }
 
   async startServerProcess(projectPath) {
-    // For some reason Atom only detects the correct env if started from the command line on Mac.
-    const env = process.platform === "darwin" ? await shellEnv() : process.env;
+    // For some reason Atom only detects the correct env if started from the command line on Mac and Debian.
+    const env = process.platform === "darwin" || process.platform === "darwin" ? await shellEnv() : process.env;
```

Comment:
This doesn't change behavior.

Reply:
Ough, my bad! It was supposed to be `linux` not `darwin` in the second condition.

Reply:
Unfortunately this will re-introduce #20 😕